build.gradle

```diff
@@ -8,14 +8,31 @@ buildscript {
       'kotlin'    : '1.3.21',
   ]
   ext.deps = [
-      'kotlin'      : [
-          'gradlePlugin': "org.jetbrains.kotlin:kotlin-gradle-plugin:${versions.kotlin}",
-          'stdlib'      : "org.jetbrains.kotlin:kotlin-stdlib:${versions.kotlin}"
+      assertj_core  : 'org.assertj:assertj-core:3.9.1',
+      // We don't need the latest version of AndroidX (there are no bugs that impact what LeakCanary
+      // relies on), we're sticking a bit older because most apps will be using a more recent version
+      // and they'll automatically resolve to higher version without having to necessarily resort to a
+      // resolution strategy.
+      androidx      : [
+          annotation: 'androidx.annotation:annotation:1.0.2',
+          core: 'androidx.core:core:1.0.1',
+          fragment  : 'androidx.fragment:fragment:1.0.0',
+          test      : [
+              core    : 'androidx.test:core:1.1.0',
+              espresso: 'androidx.test.espresso:espresso-core:3.1.1',
+              rules   : 'androidx.test:rules:1.1.1',
+              runner  : 'androidx.test:runner:1.1.1',
+          ]
+      ],
+      haha          : 'com.squareup.haha:haha:2.1',
+      junit         : 'junit:junit:4.12',
+      kotlin        : [
+          gradlePlugin: "org.jetbrains.kotlin:kotlin-gradle-plugin:${versions.kotlin}",
+          stdlib      : "org.jetbrains.kotlin:kotlin-stdlib:${versions.kotlin}"
       ],
       mockito       : 'org.mockito:mockito-core:2.7.5',
       mockito_kotlin: 'com.nhaarman:mockito-kotlin-kt1.1:1.5.0',
-      assertj_core: 'org.assertj:assertj-core:3.9.1',
-      junit: 'junit:junit:4.12',
+      robolectric   : 'org.robolectric:robolectric:4.0-alpha-3'
```

Comment:
FWIW, updating to latest java-code-styles would have fixed this.  see https://github.com/square/java-code-styles/pull/56.

Reply:
Nice!